英語版だとカードサイズをはみ出してしまい文字が読めない

<img width="1260" alt="Image" src="https://github.com/user-attachments/assets/0267af07-4c07-43f7-8606-ee13d0fad607" />
英語版だとカードサイズをはみ出してしまい文字が読めない

日本語だときちんとカードのサイズに収まる

@claude
フォーマットエラーがかかってるからプッシュ前にmake formatを実行してからね

1s
Run npm run format:check

> landingpage@0.[1](https://github.com/susumutomita/DogeDigger/actions/runs/15454738580/job/43504656091?pr=13#step:6:1).0 format:check
> prettier --check .

Checking formatting...
[warn] src/components/LanguageSwitcher.tsx
[warn] src/components/Navigation.tsx
[warn] Code style issues found in 2 files. Run Prettier with --write to fix.
Error: Process completed with exit code 1.

--- a/landingpage/src/components/FeaturesSection.tsx
+++ b/landingpage/src/components/FeaturesSection.tsx
@@ -103,10 +103,10 @@ export default function FeaturesSection() {
                 ) : (
                   <>
                     <div className="text-4xl md:text-5xl mb-4">{feature.icon}</div>
-                    <h3 className="text-xl md:text-2xl font-bold mb-3">
+                    <h3 className="text-xl md:text-2xl font-bold mb-3 line-clamp-2">
                       {t(`features.${feature.id}.title`)}
                     </h3>
-                    <p className="text-gray-600 dark:text-gray-300 mb-4 flex-grow">
+                    <p className="text-gray-600 dark:text-gray-300 mb-4 flex-grow line-clamp-3">
                       {t(`features.${feature.id}.description`)}
                     </p>
 
@@ -114,7 +114,7 @@ export default function FeaturesSection() {
                     <div
                       className={`overflow-hidden transition-all duration-300 ${hoveredFeature === feature.id ? 'max-h-32 opacity-100' : 'max-h-0 opacity-0'}`}
                     >
-                      <p className="text-sm text-gray-500 dark:text-gray-400">
+                      <p className="text-sm text-gray-500 dark:text-gray-400 line-clamp-4">
                         {t(`features.${feature.id}.details`)}
                       </p>
                     </div>

--- a/landingpage/src/components/PricingSection.tsx
+++ b/landingpage/src/components/PricingSection.tsx
@@ -92,8 +92,8 @@ export default function PricingSection() {
                 </div>
               )}
 
-              <h3 className="text-2xl font-bold mb-2">{t(`pricing.plans.${plan.id}.name`)}</h3>
-              <p className="text-gray-600 dark:text-gray-400 mb-4">
+              <h3 className="text-2xl font-bold mb-2 line-clamp-1">{t(`pricing.plans.${plan.id}.name`)}</h3>
+              <p className="text-gray-600 dark:text-gray-400 mb-4 line-clamp-2">
                 {t(`pricing.plans.${plan.id}.description`)}
               </p>
 
@@ -108,7 +108,7 @@ export default function PricingSection() {
                 {plan.features.map((feature, featureIndex) => (
                   <li key={featureIndex} className="flex items-start">
                     <Check className="w-5 h-5 text-[#4ECDC4] mr-2 flex-shrink-0 mt-0.5" />
-                    <span className="text-sm">
+                    <span className="text-sm line-clamp-2">
                       {t(`pricing.plans.${plan.id}.features.${featureIndex}`)}
                     </span>
                   </li>

